{
	"about": "https://jvndb.jvn.jp/ja/contents/2019/JVNDB-2019-002358.html",
	"title": "UltraVNC におけるバッファエラーの脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2019/JVNDB-2019-002358.html",
	"description": "UltraVNC には、バッファエラーの脆弱性が存在します。",
	"identifier": "JVNDB-2019-002358",
	"references": [
		{
			"text": "https://jvn.jp/vu/JVNVU97501786/",
			"source": "JVN",
			"id": "JVNVU#97501786"
		},
		{
			"text": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-8276",
			"source": "CVE",
			"id": "CVE-2019-8276"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2019-8276",
			"source": "NVD",
			"id": "CVE-2019-8276"
		},
		{
			"text": "https://www.us-cert.gov/ics/advisories/icsa-20-161-06",
			"source": "ICS-CERT ADVISORY",
			"id": "ICSA-20-161-06"
		},
		{
			"text": "https://ics-cert.kaspersky.com/advisories/klcert-advisories/2019/03/01/klcert-19-023-ultravnc-stack-based-buffer-overflow/",
			"source": "関連文書",
			"id": "KLCERT-19-023: UltraVNC Stack-based Buffer Overflow"
		},
		{
			"text": "https://jvndb.jvn.jp/ja/cwe/CWE-119.html",
			"id": "CWE-119",
			"title": "バッファエラー(CWE-119)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:uvnc:ultravnc",
			"version": "2.2",
			"vendor": "UVNC",
			"product": "Ultra VNC"
		}
	],
	"cvss": [
		{
			"version": "2.0",
			"score": "5.0",
			"type": "Base",
			"severity": "Medium",
			"vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
		},
		{
			"version": "3.0",
			"score": "7.5",
			"type": "Base",
			"severity": "High",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"date": "2020-06-11T17:28+09:00",
	"issued": "2019-04-08T12:09+09:00",
	"modified": "2020-06-11T17:28+09:00"
}
